css/styles.css

```diff
@@ -20,21 +20,49 @@ li {
 
 .pokedex-list {
   overflow: scroll;
-  font-size: 12.5px;
+  font-size: 1em;
   position: absolute;
   top: 0;
   margin-top: 40px;
-  max-width: 80%;
-  max-height: 90%;
+  width: 80%;
+  height: 90%;
   padding: 0;
-  width: 100%;
   background-color: rgba(255, 198, 198, 0.381);
 }
 
 .selected {
   background-color: #007bffa8;
 }
 
+.search-container {
+  justify-content: space-between;
+  display: flex;
+  width: 100%;
+  height: 4%;
```

Comment:
Not sure if you need to use height here. 

Check out the height section for tips: https://lms.learningfuze.com/code-guides/Learning-Fuze/curriculum/effective-css